common/src/main/java/com/microsoft/identity/common/internal/migration/TokenCacheItemMigrationAdapter.java

```diff
@@ -433,27 +433,33 @@ public static ADALTokenCacheItem findFrt(
      */
     @NonNull
     public static String getScopesForTokenRequest(@NonNull final String v1Resource) {
-        final String methodName = ":getScopesForTokenRequest";
-
-        String scopes = v1Resource;
-
-        if (!v1Resource.endsWith("/")) {
-            scopes += "/.default";
-        } else {
-            Logger.warn(
-                    TAG + methodName,
-                    "Redirect already contains postfixed \"/\" - not reappending."
-            );
-
-            scopes += ".default";
-        }
+        String scopes = getScopeFromResource(v1Resource);
 
         // Add the default scopes, as they will not be present
         scopes += " openid profile offline_access";
 
         return scopes;
     }
 
+    /**
+     * Given a v1 resource uri, append '/.default' to convert it to a v2 scope.
+     *
+     * @param resource The v1 resource uri.
+     * @return The v1 resource uri as a scope.
+     */
+    @NonNull
+    public static String getScopeFromResource(@NonNull final String resource) {
```

Comment:
We can another duplicate here : https://github.com/AzureAD/microsoft-authentication-library-common-for-android/blob/dev/common/src/main/java/com/microsoft/identity/common/internal/request/AdalBrokerRequestAdapter.java#L309

We can remove either of them and use one.

Reply:
Thanks for calling this out -- this change will need to happen in a separate PR, since this file doesn't exist in `common` in this branch